docs/introduction/recipes.md

```diff
@@ -0,0 +1,38 @@
+# Recipes
+
+Recipes are for:
+* documenting how to do something,
+* generating plans for people to do it together,
+* providing signals for coordinating their work.
+
+Recipes contain all the info required to create a resource.  In ERP (Enterprise Resource Planning) terms, it’s a combination of bills of material and routings and suppliers.
```

Comment:
"Create a resource or provide a service"
not all recopies have to result in *produce* flow, some can provide a service and make resource available for usage

Reply:
How about "to create a resource or provide a service"?  I don't understand about making a resource available for usage, that is just creating the resource, right?  If it is available, that could be an intent, but doesn't need a recipe?

Reply:
We rent bungalow from organization running an orphanage. They actually need to execute a small process before each rental. clean the bungalow, issue keys, read water meters, install new tank of gas, sometimes bring fridge and stove etc. This mostly requires some work, for example scheduling work of people doing the maintenance.

Reply:
Got it.  (And missed your initial suggestion somehow, sorry.)